Dockerfile

```diff
@@ -51,7 +51,12 @@ USER d-user
 RUN cd /sandbox && for package_name in \
 		mir-algorithm \
 		mir-random \
+		mir-core \
+		mir-runtime \
+		mir-optim \
 		mir \
+		asdf \
+		stdx-allocator \
```

Comment:
Why this? The logic is smart enough to fetch dependencies.

Reply:
A user may want to use a recent version a package, while a fetched dependency may not be recent.

Reply:
Fair enough.